opm/material/fluidsystems/blackoilpvt/LiveOilPvt.hpp

```diff
@@ -166,9 +165,9 @@ class LiveOilPvt
         }
 
         vapPar2_ = 0.0;
-        if (deck.hasKeyword("VAPPARS")) {
-            const auto& vapParsKeyword = deck.getKeyword("VAPPARS");
-            vapPar2_ = vapParsKeyword.getRecord(0).getItem("OIL_DENSITY_PROPENSITY").template get<double>(0);
+        const auto& oilvap = eclState.getSchedule().getOilVaporizationProperties(0);
```

Comment:
See previous comment - downstream should ideally not have to deal with the fact that it's controlled by VAPPARS at all, that's for parser to deal with.